nepomuk/pom.xml

```diff
@@ -13,8 +13,8 @@
         <org.slf4j-version>1.7.5</org.slf4j-version>
         <org.springframework-version>4.3.2.RELEASE</org.springframework-version>
         <spring-security-version>4.1.3.RELEASE</spring-security-version>
-        <geco.requests.version>0.13</geco.requests.version>
-        <geco.util.version>0.6</geco.util.version>
+        <geco.requests.version>0.14-SNAPSHOT</geco.requests.version>
+        <geco.util.version>0.7-SNAPSHOT</geco.util.version>
```

Comment:
request is published as 0.15. Please don't reopen until the changes are published, since this can't be merged or deployed until the correct version numbers are added (or reopen as a draft).